test/route-recognizer.spec.js

```diff
@@ -1,53 +1,53 @@
-const dynamicRoute = {'path': 'dynamic/:id','handler': {'name': 'dynamic'}};
-const multiNameRoute = {'path': 'static','handler': {'name': ['static-multiple', 'static-multiple-alias']}};
+const staticRoute = { 'path': 'static', 'handler': { 'name': 'static' } };
+const dynamicRoute = { 'path': 'dynamic/:id', 'handler': { 'name': 'dynamic' } };
+const multiNameRoute = { 'path': 'static', 'handler': { 'name': ['static-multiple', 'static-multiple-alias'] } };
 
 const routeTestData = [{
-    title: 'empty path routes',
-    route: {'path': '','handler': {'name': 'static'}},
-    isDynamic: false,
-    path: '/',
-    params: {}
-  }, {
-    title: 'static routes',
-    route: staticRoute,
-    isDynamic: false,
-    path: '/static',
-    params: {}
-  }, {
-    title: 'dynamic routes',
-    route: dynamicRoute,
-    isDynamic: true,
-    path: '/dynamic/test',
-    params: { id: 'test' }
-  }, {
-    title: 'multi-segment dynamic routes',
-    route: { 'path': 'dynamic/:id/:other','handler': { 'name': 'dynamic' }},
-    isDynamic: true,
-    path: '/dynamic/foo/bar',
-    params: { id: 'foo', other: 'bar' }
-  }, {
-    title: 'duplicate dynamic routes',
-    route: { 'path': 'dynamic/:id/:id','handler': { 'name': 'dynamic' }},
-    isDynamic: true,
-    path: '/dynamic/foo/foo',
-    params: { id: 'foo' }
-  }, {
-    title: 'star routes',
-    route: { 'path': 'star/*path','handler': { 'name': 'star' }},
-    isDynamic: true,
-    path: '/star/test/path',
-    params: { path: 'test/path' }
-  }, {
-    title: 'dynamic star routes',
-    route: { 'path': 'dynamic/:id/star/*path','handler': { 'name': 'star' }},
-    isDynamic: true,
-    path: '/dynamic/foo/star/test/path',
-    params: { id: 'foo', path: 'test/path' }
-  }];
+  title: 'empty path routes',
+  route: { 'path': '', 'handler': { 'name': 'static' } },
+  isDynamic: false,
+  path: '/',
+  params: {}
+}, {
+  title: 'static routes',
+  route: staticRoute,
+  isDynamic: false,
+  path: '/static',
+  params: {}
+}, {
+  title: 'dynamic routes',
+  route: dynamicRoute,
+  isDynamic: true,
+  path: '/dynamic/test',
+  params: { id: 'test' }
+}, {
+  title: 'multi-segment dynamic routes',
+  route: { 'path': 'dynamic/:id/:other', 'handler': { 'name': 'dynamic' } },
+  isDynamic: true,
+  path: '/dynamic/foo/bar',
+  params: { id: 'foo', other: 'bar' }
+}, {
+  title: 'duplicate dynamic routes',
+  route: { 'path': 'dynamic/:id/:id', 'handler': { 'name': 'dynamic' } },
+  isDynamic: true,
+  path: '/dynamic/foo/foo',
+  params: { id: 'foo' }
+}, {
+  title: 'star routes',
+  route: { 'path': 'star/*path', 'handler': { 'name': 'star' } },
+  isDynamic: true,
+  path: '/star/test/path',
+  params: { path: 'test/path' }
+}, {
+  title: 'dynamic star routes',
+  route: { 'path': 'dynamic/:id/star/*path', 'handler': { 'name': 'star' } },
+  isDynamic: true,
+  path: '/dynamic/foo/star/test/path',
+  params: { id: 'foo', path: 'test/path' }
+}];
```

Comment:
Please make whitespace changes in a separate PR so it's easier to see the functional changes made.